website/docs/r/pi_network_security_group_rule.html.markdown

```diff
@@ -0,0 +1,137 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_pi_network_security_group_rule"
+description: |-
+  Manages pi_network_security_group_rule.
+subcategory: "Power Systems"
+---
+
+# ibm_pi_network_security_group_rule
+
+Add or remove a network security group rule.
+
+## Example Usage
+
+```terraform
+    resource "ibm_pi_network_security_group_rule" "network_security_group_rule" {
+      pi_cloud_instance_id         = "<value of the cloud_instance_id>"
+      pi_network_security_group_id = "<value of network_security_group_id>"
+      pi_action                    = "allow"
+      pi_destination_ports {
+        minimum = 1200
+        maximum = 37466
+      }
+      pi_source_ports {
+        minimum = 1000
+        maximum = 19500
+      }
+      pi_protocol {
+        icmp_types = []
```

Comment:
This argument needs to be updated everywhere it is present in the documentation, including no longer having to provide an empty array.

Reply:
Updated